### ISSUE TITLE
Create lightning payment request

### ISSUE THREAD

**Issue by holzeis** [get10101/10101#178]
![Image](https://user-images.githubusercontent.com/382048/221575904-38ff1cf7-ca8b-4c69-82b3-79cc0948ac86.png)

**Acceptance Criterias**

* Upon clicking fund wallet the "create lightning payment request" workflow is opened. (we will introduce the wallet selection at a later stage)
* Use the device keypad
* format number with commas on the thousands
* Only add the copy button, skip share and ... button (focus of another ticket)
* Done should return to the wallet overview (and trigger a refresh wallet)

**Comment by da-kami** [comment]
I understand that this ticket is a subset of #71 - it would be great if you could update #71 to reflect the remaining part once this ticket is done!
